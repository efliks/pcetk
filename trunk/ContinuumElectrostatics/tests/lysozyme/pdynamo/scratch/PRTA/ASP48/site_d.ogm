ON_GEOM_CENT 121 2.000000
(15.054800 22.821900 29.905363) 101 1.000000
(15.054800 22.821900 29.905363) 101 0.500000
(15.054800 22.821900 29.905363) 101 0.250000
