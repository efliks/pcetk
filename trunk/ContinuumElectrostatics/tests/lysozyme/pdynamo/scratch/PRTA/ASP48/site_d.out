Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/ASP48//site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.664701, 20.5141, 19.0512)) 121 2
Finer level:    (15.0548, 22.8219, 29.9054) 101 1
Finer level:    (15.0548, 22.8219, 29.9054) 101 0.5
Finest level:   (15.0548, 22.8219, 29.9054) 101 0.25


Self energy of scratch/PRTA/ASP48//site_d in three dielectric environment = 305.166
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/ASP48//site_d with background charge set in three dielectric environment = -4.00105
(probably in kcal/mole)



Interaction energies of scratch/PRTA/ASP48//site_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.00452066
0   1 : -6.41513e-05
1   0 : 1.53768e-06
1   1 : 0.00477467
2   0 : -0.00242563
2   1 : -5.05423e-05
3   0 : -0.00964371
3   1 : -0.00101773
3   2 : 0.000489757
3   3 : 0.00841706
4   0 : -0.00102585
4   1 : 0.00510413
5   0 : -0.00312814
5   1 : 0.0192665
6   0 : 0.00360517
6   1 : 0.0378374
7   0 : -0.0182268
7   1 : -0.000470151
8   0 : -0.00189439
8   1 : 0.276104
9   0 : 399.993
9   1 : 610.325
10   0 : 0.0489554
10   1 : 0.671332
11   0 : -0.0523304
11   1 : 0.21363
12   0 : 0.0438068
12   1 : 0.398109
13   0 : -0.000432795
13   1 : 0.00657157
14   0 : -0.0134305
14   1 : -0.000599452
15   0 : -0.0249292
15   1 : -3.68068e-05
16   0 : -0.00554874
16   1 : 0.113586
17   0 : -0.0438643
17   1 : 0.000421663
18   0 : 8.1562e-05
18   1 : 0.00614763
Total runtime: 16.46 s user 0.15 s system 
